components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -1122,30 +1120,6 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
                 context = WithDefaultConfig.mark(context);
             }
 
-            // if this is an explicit prebuild, check if the user wants to install an app.
```

Comment:
This is no longer supported for quite a while, i.e. `context` may never get parsed to `StartPrebuildContext` and reach `createWorkspace` here.

`StartPrebuildContext` is used to encapsulate a rich context in prebuild manager, which then uses this as input for workspace starter.

Reply:
> I'm missing the context atm to verify this

That can be taken quite literally ;-) 

`https://gitpod.io/new/#prebuild/https://github.com/gitpod-io/gitpod`

☝🏻 this results in that 👇🏻 

```
We're sorry, there seems to have been an error.
Error: Couldn't parse context 'prebuild/https://github.com/gitpod-io/gitpod'
```

But in `createWorkspace` the cold path was never removed. 🤷🏻‍♂️

Reply:
Quick test on the preview env: 
https://at-remove-f7bcae44fe.preview.gitpod-dev.com/new/#https://github.com/AlexTugarev/bel ✔️ (workspace is starting fine)